site/mkdocs.yml

```diff
@@ -49,8 +49,9 @@ nav:
   - User docs:
     - Getting Started: getting-started.md
     - Spark: spark.md
-    - Spark Structured Streaming: spark-structured-streaming.md
+    - Spark Streaming: spark-structured-streaming.md
```

Comment:
I think it's important for Spark users to "specify" whether it is DStream or Structured Streaming. They often mixed up both which brings confusion.

Reply:
I think it's okay to use the short form in menu navigation. This doesn't change the actual page title, right?

Reply:
Confirmed locally that it doesn't change the page title. Are you okay using a shorter name in the nav?

Reply:
Ah OK that's short form on the menu. Looks great.